packages/fxa-auth-server/lib/payments/google-play/subscriptions.ts

```diff
@@ -63,4 +74,23 @@ export class PlaySubscriptions
 
     return purchases.map(abbrevPlayPurchaseFromSubscriptionPurchase);
```

Comment:
Yes I agree with you it would be nice to refactor, but I would prefer to defer that to another patch, since those methods were not created in this PR, and we now have a few routes (not just /account) that leverage them. I can file a maintenance ticket for it.

Reply:
I filed https://mozilla-hub.atlassian.net/browse/FXA-4216 for the refactor.